docs/docsite/rst/inventory_guide/intro_patterns.rst

```diff
@@ -174,11 +174,36 @@ you can use subscripts to select individual hosts or ranges within the webserver
 
     webservers[0]       # == cobweb
     webservers[-1]      # == weber
-    webservers[0:2]     # == webservers[0],webservers[1]
-                        # == cobweb,webbing
+    webservers[0:2]     # == webservers[0],webservers[1],webservers[2]
+                        # == cobweb,webbing,weber
+    webservers[1:2]     # == webservers[1],webservers[2]
+                        # == webbing,weber
     webservers[1:]      # == webbing,weber
     webservers[:3]      # == cobweb,webbing,weber
 
++--------------------------+--------------------------------+----------+
+| Operation                | Result                         | Notes    |
++==========================+================================+==========+
+| ``s[i]``                 | *i*\ th item of *s*, origin 0  | \(1)     |
++--------------------------+--------------------------------+----------+
+| ``s[i:j]``               | slice of *s* from *i* to *j*   | \(2)     |
++--------------------------+--------------------------------+----------+
+
```

Comment:
Avoid putting footnotes in tables because it makes navigation _really_ difficult for users with assistive technology.

I feel like a decent alternative here is to put things under different headings. Not sure what the correct heading level is for this part of the guide so I'll just use markdown in this comment:

= Slicing operations

Use slicing operations to extract text that matches the regex pattern.

== Slicing at specific items

* **Operation:** ``s[i]``
* **Result:** ``i-th`` item of ``s`` where the indexing origin is ``0``

If *i* is negative, the index is relative to the end of sequence *s*: ``len(s) + i`` is substituted. However ``-0`` is is ``0``.

== Slicing with start and end points

* **Operation:** ``s[i:j]``
* **Result:** slice of ``s`` from ``i`` to ``j`` 

The slice of *s* from *i* to *j* is defined as the sequence of items with index
   *k* such that ``i <= k <= j``. If *i* is omitted, use ``0``.
   If *j* is omitted, use ``len(s)``.
   The slice omitting both *i* and *j*, results in an invalid host pattern.
   If *i* is greater than *j*, the slice is empty.
   If *i* is equal to *j*, the *s[i]* is substituted.